app/services/iiif_print/manifest_builder_service_behavior.rb

```diff
@@ -75,13 +75,16 @@ def sanitize_v3(hash:, presenter:, hits:)
 
     def apply_metadata_to_canvas(canvas:, presenter:, hits:)
       return if @child_works.empty?
+
       # uses the 'id' property for v3 manifest and `@id' for v2, which is a URL that contains the FileSet id
       file_set_id = (canvas['id'] || canvas['@id']).split('/').last
       # finds the image that the FileSet is attached to and creates metadata on that canvas
-      image = hits.find { |doc| doc[:member_ids_ssim]&.include?(file_set_id) }
+      image = hits.find { |hit| hit[:member_ids_ssim]&.include?(file_set_id) }
```

Comment:
I don't mind the use of doc here, because it lets me know we're dealing with a solr doc. 

So I kind of wish the hits param was renamed to solr_doc_hits or something, but that may impact various other places so this comment isn't a blocker.